test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,663 @@
+
+from marvin.cloudstackTestCase import cloudstackTestCase
+#from marvin.cloudstackTestCase import *
```

Comment:
Please remove code, do not comment it. It can be retrieved if necessary from Git.